qutebrowser/utils/standarddir.py

```diff
@@ -147,9 +147,12 @@ def _init_data(args: Optional[argparse.Namespace]) -> None:
     # system_data
     _locations.pop(_Location.system_data, None)  # Remove old state
     if utils.is_linux:
-        path = '/usr/share/' + APPNAME
-        if os.path.exists(path):
-            _locations[_Location.system_data] = path
+        if version.is_flatpak():
+            path = '/app/share/' + APPNAME
+        else:
+            path = '/usr/share/' + APPNAME
+    if os.path.exists(path):
+        _locations[_Location.system_data] = path
```

Comment:
This should still be under `if utils.is_linux`, no?

```suggestion
        if os.path.exists(path):
            _locations[_Location.system_data] = path
```

Reply:
Right! Sorry for that.